src/pacasam/connectors/lipac.py

```diff
@@ -90,14 +91,16 @@ def download_database(self, extraction_sql_query: str, max_chunksize_for_postgis
         gdf = gdf.set_crs(self.lambert_93_crs)
         gdf = gdf.sort_values(by=PATCH_ID_COLNAME)
         gdf = gdf.drop_duplicates(subset=PATCH_ID_COLNAME)
+        gdf[PATCH_ID_COLNAME] = gdf[[PATCH_ID_COLNAME, FILE_ID_COLNAME]].apply(
+            lambda row: f"{row[FILE_ID_COLNAME]}-{str(row[PATCH_ID_COLNAME]).zfill(ZFILL_MAX_PATCH_NUMBER)}", axis=1
+        )
         gdf[FILE_PATH_COLNAME] = gdf[FILE_PATH_COLNAME].apply(self.convert_samba_path_to_mounted_path)
         return gdf
 
     def convert_samba_path_to_mounted_path(self, samba_path):
         """Convert Samba path to its mounted path, expected to be under /mnt/store-lidarhd/."""
-        relative_path = PureWindowsPath(samba_path).relative_to(PureWindowsPath("//store.ign.fr"))
-        mounted_path = Path(self.mounted_store_path) / relative_path
-        return mounted_path
+        mounted_path = PureWindowsPath(samba_path).as_posix().replace("//store.ign.fr", self.mounted_store_path)
```

Comment:
what was the issue with the former implementation? (Its looks to me that it is safer to use `relative_to` to prevent issues like upper case names for example)

Reply:
I'm not sure why, with the former implementation I got an error saying that the samba_path was not relative to the store. That's why I had to update it, although it is less elegant to manipulate strings.

The expected samba_path is pretty constrainted. I don't see a situation where one method is safer that the other, do you have one in mind?

Reply:
![image](https://github.com/IGNF/pacasam/assets/11660435/1425dee4-3214-44df-8d5c-9daf035e7176)
Some strange regression occured where `//store.ign.fr/store-lidarhd/` is considered as a single folder.
A fix could be to use this and then adding back the store-lidarhd part:
  relative_path = PureWindowsPath(samba_path).relative_to("//store.ign.fr/store-lidarhd/")
  mounted_path = Path(self.mounted_store_path) / "store-lidarhd" / relative_path

But this seems overkill compared to a simple replace and not really needed since we will always have lowercase //store.ign.fr. I'm keeping this as it is, it's not worth the investigation.